contracts/multiswap-contracts/FundManager.sol

```diff
@@ -346,11 +632,73 @@ contract FundManager is SigCheckable, WithAdmin, TokenReceivable {
         return actualLiq;
     }
 
+    /**
+     * @dev Retrieves liquidity for the specified token and liquidity adder.
+     * @param token Token address for liquidity.
+     * @param liquidityAdder Address of the liquidity adder.
+     * @return Current liquidity amount.
+     */
     function liquidity(address token, address liquidityAdder)
         external
         view
         returns (uint256)
     {
         return liquidities[token][liquidityAdder];
     }
+
+    /**
+     * @dev Adds liquidity for the specified token, managed exclusively by liquidity managers.
```

Comment:
We are getting rid of `addLiquidityByManager` because we can simply send tokens to the contract to refill it.